crates/host/src/wasmbus/ctl.rs

```diff
@@ -986,7 +1002,7 @@ impl ControlInterfaceServer for Host {
             .uptime_seconds(uptime.as_secs())
             .uptime_human(human_friendly_uptime(uptime))
             .version(self.host_config.version.clone())
-            .ctl_host(self.host_config.ctl_nats_url.to_string())
+            .ctl_host(self.host_config.rpc_nats_url.to_string())
```

Comment:
Nit: Might be worth a comment here to note that we removed the `ctl_nats_url` from the host config so people don't get tripped up later when they don't match